src/view.py

```diff
@@ -56,57 +56,98 @@ def set_settings(self, values, path):
         #print(path)
         #print(type(path))
 
-
+        valid = True
+        error_list = list([])
 
         #all possible errors
-        error_missing_keys = "ERROR: MISSING KEYS"
-        error_extra_keys = "ERROR: EXTRA KEYS"
-        error_type_error = "ERROR: TYPE ERROR"
-        error_conf_out_of_range = "ERROR: CONF OUT OF RANGE"
-        error_poll_less_than_0 = "ERROR: POLL LESS THAN 0"
-        error_anti_less_than_0 = "ERROR: ANTI LESS THAN 0"
-        error_runtime_less_than_0 = "ERROR: RUNTIME LESS THAN 0"
+        error_missing_keys = "ERROR: _ MISSING KEYS"
+        error_extra_keys = "ERROR: _ EXTRA KEYS"
+        error_no_conf = "ERROR: NO CONF"
+        error_no_poll = "ERROR: NO POLL"
+        error_no_anti = "ERROR: NO ANTI"
+        error_no_runtime = "ERROR: NO RUNTIME"
+        error_no_path = "ERROR: NO PATH"
+        error_no_term = "ERROR: NO TERM"
+        error_conf_out_of_range = "ERROR: YOUR CONF IS _, AND IT HAS TO BE BETWEEN 0 AND 1"
+        error_poll_less_than_0 = "ERROR: YOUR POLL IS _, AND IT HAS TO BE LESS THAN 0"

```

Comment:
Shouldn't this say "AND IT HAS TO BE GREATER THAN 0"?

Reply:
Yup.